Example/Extending/BrightnessCapability/BrightnessCapability.swift

```diff
@@ -0,0 +1,99 @@
+import UIKit
+import MediaEditor
+
+class BrightnessCapability: MediaEditorCapability {
+    static var name = "Brightness"
+
+    static var icon = UIImage(named: "ink")!
+
+    var image: UIImage
+
+    var viewController: UIViewController
+
+    var onFinishEditing: (UIImage, [MediaEditorOperation]) -> ()
+
+    var onCancel: (() -> ())
+
+    required init(_ image: UIImage, onFinishEditing: @escaping (UIImage, [MediaEditorOperation]) -> (), onCancel: @escaping () -> ()) {
+        self.image = image
+        self.onFinishEditing = onFinishEditing
+        self.onCancel = onCancel
+        let viewController = BrightnessViewController.fromStoryboard()
+        self.viewController = viewController
+        viewController.onFinishEditing = onFinishEditing
+        viewController.onCancel = onCancel
+        viewController.image = image
+    }
+
+    func apply(styles: MediaEditorStyles) {
+        // Apply styles here
+    }
+
+}
+
+class BrightnessViewController: UIViewController {
```

Comment:
What are your thoughts on extracting `BrightnessViewController ` into its own file? I feel like it's cluttering the example code above.

Reply:
My idea is to change the `MediaEditorCapability` so we'll reduce all of this into one entity (instead of 2).

I've added them to the same file with the assumption that this would make it easier to read — instead of making the user go through multiple files.

But I don't have a strong opinion on it.

What do you think? Split? Yes? No?

Reply:
I can see that point. I don't have a strong opinion either way, but if we keep it in the same file then I think adding a `// MARK: - BrightnessViewController` separator would help break it up.

Reply:
Done in e8008e705ee01dcd9cb46db9b06204335bf60c7e